browse/static/js/catalyzex.js

```diff
@@ -0,0 +1,59 @@
+  }
+
+  const fetchCatalyzeXCode = async () => {
+    const cxApiUrl = `https://www.catalyzex.com/api/code?src=arxiv&paper_arxiv_id=${arxivId}`;
+
+    let result = {};
+
+    try {
+      result = await $.ajax({ url: cxApiUrl, timeout: 2000, dataType: "json" });
+    } catch (error) {
+      result = {};
+    }
+
+    return result;
+  };
+
+  $output.html('');
+
+  const { count: implementations, cx_url: cxImplementationsUrl } = await fetchCatalyzeXCode()
+
+  $output.append("<h2>CatalyzeX</h2>");
+
+  const submitItHereLink = `<a target="_blank" href="https://www.catalyzex.com/add_code">submit it here</a>`;
```

Comment:
can we prefill the `add_code` link based on whatever we do have available? :)

Reply:
Absolutely. I've just updated the PR so that we can automatically prefill both `Paper Title` and `Paper URL` fields :)


https://user-images.githubusercontent.com/58868651/216734734-f21b116b-506f-4e9f-8da7-40d91c643126.mov